### DISCUSSION TITLE
Changing WiFi settings to connect to home network?

### DISCUSSION THREAD

**Discussion by Kaos116** [thehighnibble/thehighnibble.github.io#45] [Q&A] [answered]
I can't figure out how to do this. How is this done? I have made changes to the boot.conf file, but I must be missing something. Does anyone have an example of the setting to connect to a home network? I have no issues connecting to the default IMSAI8080 network.

**Comment by dmcnaugh** [comment, marked as answer]
In addition the the `boot.conf` file changes documented here: https://thehighnibble.com/imsai8080/configure/#wi-fi-communications
you also need to set the `NVS_IF_STA` bit to `1` in the **NVS Startup configuration**, see: https://thehighnibble.com/imsai8080/configure/#startup-configuration-non-volatile-storage-nvs

**Reply by Kaos116** [reply]
Thank you for the quick answer. I was missing the NVS_IF_STA bit.
